openmrs/i18n/registration/locale_fr.json

```diff
@@ -53,11 +55,12 @@
   "ADDRESSHIERARCHY_CITY":"Ville",
   "ADDRESSHIERARCHY_DISTRICT":"District",
   "REGISTRATION_IDTYPE_1_CNI_KEY":"Numéro CNI",
-  "REGISTRATION_IDTYPE_2_ART_KEY":"Numéro TARV",
-  "REGISTRATION_IDTYPE_3_ANC_KEY":"Numéro CPN",
-  "REGISTRATION_IDTYPE_4_PASSPORT_KEY":"Numéro de Passport",
-  "REGISTRATION_IDTYPE_5_RECEIPT_KEY":"Numéro de Récépissé",
-  "REGISTRATION_IDTYPE_6_OTHER_KEY":"Autre numéro",
+  "REGISTRATION_IDTYPE_2_UIN_KEY": "Numero unique CNI",
```

Comment:
@prashantsable1991 Please use "Numéro unique CNI" here

Reply:
@lauravignoli on ticket as translations provided to me I added that. I think we need to confirm all translations are added and they are correct as well before adding a ticket in "Ready for dev". I'm adding a translation provided by you. Thanks.